apps/src/templates/sectionProgress/progressTables/ProgressTableLessonNumber.jsx

```diff
@@ -85,26 +86,6 @@ const styles = {
     color: color.white,
     fontSize: 18
   },
-  arrowContainer: {
```

Comment:
For my education, what was the reason you decided to move these styles to the scss file?

Reply:
while it may be possible, i didn't know how to accomplish the `html[dir='RTL']` conditional styling using js/radium so i just went with the method i knew.